tests/managed/overloads.cs

```diff
@@ -277,4 +277,37 @@ public static AllOperatorsWithFriendly Negate (AllOperatorsWithFriendly c1)
 			return new AllOperatorsWithFriendly (-c1.Value);
 		}	
 	}
+
+	public class EqualOverrides {
+		public int Value { get; private set; }
+
+		public EqualOverrides (int val)
+		{
+			Value = val;
+		}
+
+		public static bool operator == (EqualOverrides a, EqualOverrides b)
+		{
+			return a.Value == b.Value;
+		}
+
+		public static bool operator != (EqualOverrides a, EqualOverrides b)
+		{
+			return a.Value == b.Value;
+		}
+
+		public override bool Equals (Object obj)
+		{
+			if (obj == null || GetType () != obj.GetType ())
+				return false;
+
+			EqualOverrides p = (EqualOverrides)obj;
+			return Value == p.Value;
+		}
+
+		public override int GetHashCode ()
```

Comment:
just like binding `GetHashCode` overrides the default `hash` implementation from `NSObject`